std/algorithm/sorting.d

```diff
@@ -3019,6 +3027,14 @@ if (isRandomAccessRange!R && hasLength!R && hasSwappableElements!R)
     schwartzSort!((Tuple!(char) c){ return c[0]; })(chars);
 }
 
+@safe unittest
+{
+    // issue 13965
+    import std.typecons : Tuple;
+    Tuple!(char)[] chars;
+    schwartzSort!("a[0]", SwapStrategy.stable)(chars);
```

Comment:
Can we add a more complete with actual data to ensure the ordering actually works? Thanks!